Formula/nibtools.rb

```diff
@@ -0,0 +1,25 @@
+class Nibtools < Formula
+  desc "Commodore 1541/1571 disk image nibbler"
+  homepage "https://c64preservation.com/dp.php?pg=nibtools"
+  url "https://c64preservation.com/svn/nibtools/trunk", :using => :svn, :revision => "649"
```

Comment:
We need stable, released versions: https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md